test/e2e/json-rpc/eth_sendTransaction.spec.js

```diff
@@ -68,11 +59,7 @@ describe('eth_sendTransaction', function () {
         fixtures: new FixtureBuilder()
           .withPermissionControllerConnectedToTestDapp()
           .build(),
-        localNodeOptions: generateGanacheOptions({
-          hardfork: 'london',
-          mnemonic:
-            'phrase upgrade clock rough situate wedding elder clever doctor stamp excess tent',
```

Comment:
this  mnemonic is also not needed. We have the mnemonic set in the node class level, where we directly seed the accounts